apps/genrsa.c

```diff
@@ -123,6 +124,10 @@ int genrsa_main(int argc, char **argv)
     if (argc == 1) {
         if (!opt_int(argv[0], &num) || num <= 0)
             goto end;
+        if (num > RSA_RECOMMENDED_MAX_KEY_LENGTH) 
+            BIO_printf(bio_err, "# Warning: It is not recommended to use more than %d bit for RSA keys.      #\n"
```

Comment:
Indentation: Insert linebreak before argument 2 (the literal string) and align the two literal strings with the `b` of `bio_err`.